lib/data_migrate/data_schema.rb

```diff
@@ -30,10 +30,12 @@ def migrated
 
     def versions
       @versions ||= begin
-        versions = []
-        Dir.foreach(DataMigrate::DataMigrator.full_migrations_path) do |file|
-          match_data = DataMigrate::DataMigrator.match(file)
-          versions << match_data[1].to_i if match_data
+        versions = Set.new
+        DataMigrate::DataMigrator.migrations_paths.each do |path|
```

Comment:
It looks like this might be changing the list [from an expanded path](https://github.com/trusted/data-migrate/blob/3807de412e2f91d79861716488389913387c35be/lib/data_migrate/data_migrator_five.rb#L54) to a relative path. It might be an issue if running the `data_migrate` command from somewhere other than `Rails.root` ? Pretty edge case, would not consider this a blocker for deployment, but maybe something the upstream folks will be concerned about 😸